http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/BaseCloudServiceRuntimeMonitor.scala

```diff
@@ -528,12 +539,18 @@ abstract class BaseCloudServiceRuntimeMonitor[F[_]] {
             ).map(b => (imageType, b))
           )
       }
-      // wait for 10 minutes for tools to start up before time out.
+      // wait for tools to start up before time out. Use the timeout value if available
+      // or default to 10 minutes
+      timeout: FiniteDuration = timeoutInMinutes match {
+        case Some(duration) => duration
+        case None =>
+          monitorConfig.checkTools.interruptAfter
+      }
       availableTools <- streamFUntilDone(
         checkTools,
         monitorConfig.checkTools.maxAttempts,
         monitorConfig.checkTools.interval
-      ).interruptAfter(monitorConfig.checkTools.interruptAfter).compile.lastOrError
+      ).interruptAfter(timeout).compile.lastOrError
```

Comment:
Would the default config interval and max attempts terminate at the default 10 minute timeout even if we set the interruptAfter?

Reply:
ah yea, that too...so I think we need to adjust the max attempts and interval as well if user defines timeout

Reply:
Gotcha, thanks for the catch @n8landolt 
I will implement what you had previously
https://github.com/DataBiosphere/leonardo/blob/5150e7dc0d7dda5f525a2e09fe3b12afd95da822/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/BaseCloudServiceRuntimeMonitor.scala#L474

I will also modify the corresponding unit test